tests/random.js

```diff
@@ -0,0 +1,41 @@
+/* random.js
+ *
+ * Copyright (C) 2006-2022 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+const { WolfSSLRandom } = require( '../interfaces/random' )
+
+const rng_tests =
+{
+    generateBlock: async function()
+    {
+        let rng = new WolfSSLRandom()
+        let rngOne = rng.GenerateBlock(1000)
```

Comment:
Please also check for all zeros. Consider generating smaller block or making const variable.